bugsnag-android-core/src/main/java/com/bugsnag/android/Configuration.java

```diff
@@ -6,6 +6,7 @@
 import androidx.annotation.Nullable;
 
 import java.lang.reflect.Field;
+import java.util.Arrays;
```

Comment:
related?

Reply:
An import was missing on the `immutable-config` branch (I think due to a merge conflict) which would otherwise fail the build.